icommons_ext_tools/requirements/local.txt

```diff
@@ -4,6 +4,7 @@
 -r base.txt 
 
 # below are requirements specific to the local environment
-django-debug-toolbar==1.3.2
+django-debug-toolbar==1.6
```

Comment:
As discussed offline, lets downgrade it to version 1.5 due to change in support for automatic setup